src/api/courses.js

```diff
@@ -74,7 +74,27 @@ const getCsv = questions => {
 // Get all courses
 router.get(
   '/',
-  safeAsync(async (req, res, _next) => res.send(await Course.findAll()))
+  safeAsync(async (req, res, _next) => {
+    const {
+      locals: { userAuthz },
+    } = res
+    const courses = await Course.findAll()
+    if (userAuthz.isAdmin) {
+      res.send(courses)
+    } else {
+      const coursesToShow = courses.filter(course => {
```

Comment:
Nit: would it be possible to do this filtering with the query itself, instead of reading all courses and then filtering them?